src/ocm/components/clusterConfiguration/HostInventory.tsx

```diff
@@ -67,7 +67,7 @@ const HostInventory: React.FC<{ cluster: Cluster }> = ({ cluster }) => {
   );
   const isOpenshiftClusterStorageEnabled = useFeature('ASSISTED_INSTALLER_OCS_FEATURE');
   const isContainerNativeVirtualizationEnabled = useFeature('ASSISTED_INSTALLER_CNV_FEATURE');
-  const isSNO = isSingleNodeCluster(cluster);
+  const isSNOCluster = isSNO(cluster);
```

Comment:
maybe
const isSno = isSno(cluster)?
less changes and consistent with other places

Reply:
you're right, just leave it as is